### PR TITLE
chore: update sphere information

### PR DESCRIPTION
**Pull request by 0x0sub**

### CODE REVIEW

File: pages/chain/beam-ecosystem/sphere.mdx [hunk 1]
```diff
@@ -19,12 +19,25 @@ Explore Beam’s native marketplace at [sphere.market](https://sphere.market).
...
 
 ### Are there any fees associated with Sphere transactions?
 
-Yes, selling an asset on Sphere incurs a standard fee of 2.5% of the transaction amount, charged in the currency used for the sale. Additionally, there might be royalties set by the creator and a gas fee to initiate the transaction. Learn more about [Sphere Fees](https://sphere.market/fees).
+Yes, when selling an asset on Sphere, the following fees occur:
+  - Trading fee: This is a standard 2% of the paid amount charged in the currency used to make the sale. 
+  - Royalties: This is a variable percentage that is set by the creator.
+  - Gas fees: When selling an NFT, you need to approve the smart contracts to interact with your assets, the actual transaction is covered by the buyer of the NFT.
+
+When buying an asset on Sphere only the gas fees needed to be payed.
```
Comment (royscheepens) [inline pages/chain/beam-ecosystem/sphere.mdx:31]: Payed is not the correct spelling here, can you change it to paid?

File: pages/chain/beam-ecosystem/sphere.mdx [hunk 2]
```diff
@@ -19,12 +19,25 @@ Explore Beam’s native marketplace at [sphere.market](https://sphere.market).
...
-Apart from Beam, Sphere is accessible on Avalanche, Ethereum, and Polygon networks.
+Apart from Beam, Sphere is accessible on the Immutable zkEVM network.
 
 ### Are there any fees associated with Sphere transactions?
 
-Yes, selling an asset on Sphere incurs a standard fee of 2.5% of the transaction amount, charged in the currency used for the sale. Additionally, there might be royalties set by the creator and a gas fee to initiate the transaction. Learn more about [Sphere Fees](https://sphere.market/fees).
+Yes, when selling an asset on Sphere, the following fees occur:
+  - Trading fee: This is a standard 2% of the paid amount charged in the currency used to make the sale. 
+  - Royalties: This is a variable percentage that is set by the creator.
+  - Gas fees: When selling an NFT, you need to approve the smart contracts to interact with your assets, the actual transaction is covered by the buyer of the NFT.
```
Comment (royscheepens) [inline pages/chain/beam-ecosystem/sphere.mdx:29]: You can also mention the fact that cancelling a listing on Beam also incurs gas fees. On the Immutable zkEVM chain, we support gassless cancellations.

**Review by royscheepens — CHANGES_REQUESTED** [review]

File: pages/chain/beam-ecosystem/sphere.mdx [hunk 3]
```diff
@@ -19,12 +19,25 @@ Explore Beam’s native marketplace at [sphere.market](https://sphere.market).
...
+Apart from Beam, Sphere is accessible on the Immutable zkEVM network.
 
 ### Are there any fees associated with Sphere transactions?
 
-Yes, selling an asset on Sphere incurs a standard fee of 2.5% of the transaction amount, charged in the currency used for the sale. Additionally, there might be royalties set by the creator and a gas fee to initiate the transaction. Learn more about [Sphere Fees](https://sphere.market/fees).
+Yes, when selling an asset on Sphere, the following fees occur:
+  - Trading fee: This is a standard 2% of the paid amount charged in the currency used to make the sale. 
+  - Royalties: This is a variable percentage that is set by the creator.
+  - Gas fees: When selling an NFT, you need to approve the smart contracts to interact with your assets, the actual transaction is covered by the buyer of the NFT. Cancelling a listing on Beam also incurs gas fees. However, on the Immutable zkEVM chain, we support gasless cancellations.
+
```
Comment (victorboucher) [inline pages/chain/beam-ecosystem/sphere.mdx:30]: You could also mention Immutable their fees here (2%).

**Review by victorboucher — CHANGES_REQUESTED** [review]

File: pages/chain/beam-ecosystem/sphere.mdx (hunk 3 above)
Comment (0x0sub) [reply pages/chain/beam-ecosystem/sphere.mdx:30]: Added, this might need to be added to the Sphere website too.

Comment (victorboucher) [reply pages/chain/beam-ecosystem/sphere.mdx:30]: Will do!

**Review by victorboucher — APPROVED** [review]

**Comment by 0x0sub** [comment]
@royscheepens if you have a minute, please review

**Review by royscheepens — APPROVED** [review]

**Pull request merged**